walkthroughs/cdk-examples/app-mesh-hands-on/templates/ecs-services-stack.yaml

```diff
@@ -0,0 +1,100 @@
+Description: Defines the Fargate services and their task definitions.
```

Comment:
Let's not include these auto generated files.

Reply:
Deleting the templates dir entirely